From: Mark White <mark.white@ucl.ac.uk>
Subject: bugfix for fabber

There's a small bug in the ND package fsl-5.0 causing fabber to
build without the Oxford ASL forward models; which in turn causes
the oxford_asl script to fail.  It's caused by the fabber Makefile
setting "-I${BOOSTDIR} -D__OXASL", but BOOSTDIR is undefined in the
debian build, so the -I eats the -D.

All other FSL makefiles use -I${INC_BOOST}, which is already in the
debian build environment: so I suspect it's cleanest to patch the
fabber Makefile.  The trivial patch to do so is attached.

--- a/src/fabber/Makefile	2013-03-11 16:02:59.264787529 +0000
+++ b/src/fabber/Makefile	2013-03-11 16:03:30.628784161 +0000
@@ -2,7 +2,7 @@
 
 PROJNAME = fabber
 
-USRINCFLAGS = -I${INC_NEWMAT} -I${INC_PROB} -I${BOOSTDIR} -D__OXASL
+USRINCFLAGS = -I${INC_NEWMAT} -I${INC_PROB} -I${INC_BOOST} -D__OXASL
 USRLDFLAGS = -L${LIB_NEWMAT} -L${LIB_PROB}
 
 LIBS = -lutils -lnewimage -lmiscmaths -lprob -lnewmat -lfslio -lniftiio -lznz -lz 
